src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/caseofficer/CaseOfficerSubmitTimeExtensionPersonalisation.java

```diff
@@ -43,7 +43,7 @@ public String getTemplateId() {
 
     @Override
     public Set<String> getRecipientsList(AsylumCase asylumCase) {
-        return featureToggler.getValue("tcw-notifications-feature", false)
+        return featureToggler.getValue("tcw-application-notifications-feature", false)
```

Comment:
@burale I think we should set the default value to true. This is better practice, by default the notification should go out.